packages/palette_generator/example/lib/main.dart

```diff
@@ -41,20 +41,20 @@ class MyApp extends StatelessWidget {
 class ImageColors extends StatefulWidget {
   /// Creates the home page.
   const ImageColors({
-    Key key,
+    Key? key,
     this.title,
-    this.image,
+    required this.image,
     this.imageSize,
   }) : super(key: key);
 
   /// The title that is shown at the top of the page.
-  final String title;
+  final String? title;
 
   /// This is the image provider that is used to load the colors from.
   final ImageProvider image;
 
   /// The dimensions of the image.
-  final Size imageSize;
+  final Size? imageSize;
```

Comment:
Why is this nullable?

Reply:
`ImageConfiguration`, which is a Flutter method, accept null as an argument as `size` property

https://github.com/flutter/packages/blob/9f0a365d85c9e6b3561c3d227b226f66e28b257e/packages/palette_generator/lib/palette_generator.dart#L179